toolchain/check/call.cpp

```diff
@@ -18,6 +19,15 @@
 
 namespace Carbon::Check {
 
+namespace {
+// Entity kinds for the diagnostic. Converted to an int for a select.
+enum class EntityKind : uint8_t {
+  Function,
+  GenericClass,
+  GenericInterface,
```

Comment:
```suggestion
  Function = 0,
  GenericClass = 1,
  GenericInterface = 2,
```

Since the integer values are significant, we should probably declare them explicitly.